crates/util/edcode-derive/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "rimecraft-edcode-derive"
+version = "0.1.0"
+edition = "2021"
+authors = ["C191239 <zhushunzhong2025@i.pkuschool.edu.cn>"]
+description = "Derive macros for rimecraft-edcode"
+repository = "https://github.com/rimecraft-rs/rimecraft/"
+license = "AGPL-3.0-or-later"
+categories = ["encoding"]
+
+[badges]
+maintenance = { status = "passively-maintained" }
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+syn = { version = "2.0" }
+quote = { version = "1.0" }
+proc-macro2 = { version = "1.0" }
```

Comment:
these could be simplified as `crate-name = "version"`.

Reply:
fix in next commit